resources/build/version/src/fixupHistory.ts

```diff
@@ -193,7 +193,7 @@ export const fixupHistory = async (
   let pulls: PRInformation[] = [];
 
   try {
-    pulls = await reportHistory(octokit, base, force, true);
+    pulls = await reportHistory(octokit, base, force, false);
```

Comment:
Is this change intentional?

Reply:
Yes, addresses https://github.com/keymanapp/keyman/issues/7627#issuecomment-1321264070. We will watch it and see what happens with 16.0-stable cherry-picks.